packages/remix-runtime/README.md

```diff
@@ -0,0 +1,4 @@
+# Remix Server Runtime for Netlify
+
+The Remix Server Runtime for Netlify is a serverless edge runtime for [Remix](https://remix.run) apps. It's built on top
+of [Netlify Edge Functions](https://docs.netlify.com/functions/overview/).
```

Comment:
```suggestion
of [Netlify Edge Functions](https://docs.netlify.com/edge-functions/overview/).
```
Here we can link to the Edge Functions page of the docs instead of the Functions page.

Reply:
Good catch. I thought I updated that.

Reply:
I think like I mention below, there's an issue with latest commits not appearing on GitHub or something, because this was the README before your comments. 🤔 

```
# Remix Server Runtime for Netlify

The Remix Server Runtime for Netlify is a serverless edge runtime for [Remix](https://remix.run) apps. It's built on top
of [Netlify Edge Functions](https://docs.netlify.com/edge-functions/overview/).

```